src/Graphics/src/Graphics/Platforms/Windows/PlatformCanvas.cs

```diff
@@ -13,9 +13,19 @@
 using WColors = global::Microsoft.UI.Colors;
 #endif
 
+#if MAUI_GRAPHICS_WIN2D
 namespace Microsoft.Maui.Graphics.Win2D
+#else
+namespace Microsoft.Maui.Graphics.Platform
+#endif
 {
-	public class W2DCanvas : AbstractCanvas<W2DCanvasState>, IBlurrableCanvas
+#if MAUI_GRAPHICS_WIN2D
+	[System.Obsolete("Use Microsoft.Maui.Graphics.Platform.PlatformCanvas instead.")]
+	public class W2DCanvas
+#else
+	public class PlatformCanvas
+#endif
+		: AbstractCanvas<PlatformCanvasState>, IBlurrableCanvas
```

Comment:
Same code file, different types. Old type is obsolete, new type is named correctly.